tests/test_location/test_create_location.py

```diff
@@ -46,3 +49,45 @@ def test_create_location_that_already_exists(self):
             self,
             create_duplicate_location_query,
             'Kampala Location already exists')
+
+    def test_database_connection_error(self):
+        """
+        test a user friendly message is returned to a user when database
+        cannot be reached
+        """
+        BaseTestCase().tearDown()
+        CommonTestCases.admin_token_assert_in(
+            self,
+            create_location_query,
+            "The database cannot be reached"
+            )
+
+    def test_location_creation_with_invalid_country(self):
+        """
+        Testing for location creation
```

Comment:
There is another test for location creation elsewhere, what are we exactly testing for here. Kindly be specific in your docstring descriptions

Reply:
Rectified @Bonifase